extensions-core/azure-extensions/src/test/java/org/apache/druid/storage/azure/AzureCloudBlobIteratorTest.java

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.druid.storage.azure;
 
+import com.google.api.client.util.Lists;
```

Comment:
You'll need to update the pom to add this dependency:
```
<dependency>
  <groupId>com.google.http-client</groupId>
  <artifactId>google-http-client</artifactId>
  <scope>test</scope>
</dependency>
```
https://travis-ci.org/apache/druid/jobs/657595721#L2090

Reply:
Ah, this was a mistake. I'm not sure why the Intellij keeps adding a wrong one. Fixed it now.